sirepo/template/radia.py

```diff
@@ -1047,7 +1057,7 @@ def _validate_objects(objects):
         if 'material' in o and o.material in _SCHEMA.constants.anisotropicMaterials:
             if numpy.linalg.norm(sirepo.util.split_comma_delimited_string(o.magnetization, float)) == 0:
                 raise ValueError(
-                    '{}: anisotropic material {} requires non-0 magnetization'.format(
+                    'name={}, : material={}: anisotropic material requires non-0 magnetization'.format(
```

Comment:
I think we've settled on `f''` strings in stead of `.format()`

Reply:
My understanding was to avoid f strings in error messages but maybe I misconstrued that. I'll leave it as-is until I get clarity on the matter

Reply:
format and f-strings are identical.

We avoid f-strings and format in log messages. The assumption is that log messages could contain large values that need to be truncated by pkdebug whereas error messages are not that way. If you expect an error message to be large, then you will want to truncate manually (possible with f-strings or format, e.g. `{big_thing:.100s}`).

Reply:
You don't want to use f-strings in pkdebug functions (pkdp, pkdlog, etc). In those functions we [modify args and kwargs](https://github.com/radiasoft/pykern/blob/master/pykern/pkdebug.py#L562) (ex redact secrets). So, we need directs access to args and kwargs. In ValueError (or any other stdlib error) you can use them since we don't do any manipulation.